src/System Application/App/Permission Sets/src/PermissionSet.Page.al

```diff
@@ -177,6 +177,7 @@ page 9855 "Permission Set"
                             exit;
 
                         AddLoggedPermissions(TempTablePermissionBuffer);
+                        Session.LogAuditMessage(StrSubstNo(PermissionSetModifiedLbl, Rec."Role ID", UserSecurityId()), SecurityOperationResult::Success, AuditCategory::ApplicationManagement, 2, 0);
```

Comment:
Administred role and probably 0 (Environment) as above

Reply:
AuditCategory changed, AuditMessageOperation kept as per MS suggestion.